src/linux_joystick.c

```diff
@@ -38,6 +38,10 @@
 #include <string.h>
 #include <unistd.h>
 
+#ifndef SYN_DROPPED // < 2.6.39 kernel headers
```

Comment:
Also mention which dist and version requires this and when support for it ends so this workaround can be removed.